.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,128 @@
+name: continuous-integration
+
+# TODO: build master branch after PRs are merged.
+on:
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
Could you document the syntax?

Reply:
The syntax for the whole file and specific lines was linked to. Good idea.